### PR TITLE
4032 - Collaborators Grouped UI

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

**Comment by sorja** [comment]
Looks good
@yaguzmang Please approve when you have read it

File: tsconfig.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "allowSyntheticDefaultImports": true,
     "baseUrl": "./src",
     "isolatedModules": true,
+    "lib": ["ESNext", "DOM", "DOM.Iterable"],
```
Comment (sorja) [inline tsconfig.json:9]: What's this?

Comment (minotogna) [reply tsconfig.json:9]: to use Object.groupBy

**Review by yaguzmang — APPROVED** [review]
👍

**Pull request merged**